test/hotspot/jtreg/compiler/codecache/CodeCacheFullCountTest.java

```diff
@@ -57,7 +57,12 @@ public static void runTest() throws Throwable {
         ProcessBuilder pb = ProcessTools.createJavaProcessBuilder(
           "-XX:ReservedCodeCacheSize=2496k", "-XX:-UseCodeCacheFlushing", "-XX:-MethodFlushing", "CodeCacheFullCountTest", "WasteCodeCache");
         OutputAnalyzer oa = ProcessTools.executeProcess(pb);
-        oa.shouldHaveExitValue(0);
+        if (oa.getExitValue() != 0) {
+            // Ignore adapter creation failures
+            if (!oa.getStderr().contains("Out of space in CodeCache for adapters")) {
+                throw new Exception("VM finished with exit code " + oa.getExitValue());
```

Comment:
Please use `RuntimeException` like the code below does. I think you could also merge the two ifs.

Reply:
Hello Tobias,

Thank you for pointing it out.
I have modified some points you had mentioned.

Thanks,
Kimura Yukihiro